src/core/client/stream/classes.ts

```diff
@@ -434,16 +434,22 @@ const CLASSES = {
   },
 
   /**
-   * rejectedTombstone is shown when a comment got rejected.
+   * moderationRejectedTombstone is shown to the moderator when a comment got rejected.
    */
-  rejectedTombstone: {
-    $root: "coral coral-rejectedtombstone",
-    goToModerateButton: "coral coral-rejectedtombstone-goToModerateButton",
+  moderationRejectedTombstone: {
+    $root: "coral coral-moderationRejectedTombstone",
+    goToModerateButton:
+      "coral coral-moderationRejectedTombstone-goToModerateButton",
   },
+  /**
+   * rejectedTombstone is shown when a comment has been rejected.
+   */
+  rejectedTombstone: "coral coral-rejectedTombstone",
+
   /**
    * deletedTombstone is shown when a comment has been deleted.
    */
-  deletedTombstone: "coral coral-rejectedtombstone",
+  deletedTombstone: "coral coral-deletedTombstone",
```

Comment:
If we're going to be changing the static CSS names for these, we need to make sure we let clients know and update any custom CSS we are in charge of writing for SBN

Reply:
This is a bug though, as you can see by the naming. I hope we didn't rely on this 😱